openapi/shared/errors/unauthorized-error.yaml

```diff
@@ -0,0 +1,12 @@
+description: Incorrect username or password
```

Comment:
É melhor ser uma mensagem mais genérica, como "not authenticated ou unauthorized to use this resource"

Nem todo login é com e-mail e senha

Reply:
Okk